src/components/main/CategoryController.jsx

```diff
@@ -5,6 +5,15 @@ import ArticleList from 'components/main/ArticleList';
 import NotFound from 'components/main/NotFound';
 import _ from 'lodash';
 
+const uppercase = str => {
```

Comment:
Could you add a comment explaining this section?

Reply:
I just moved the function to a higher scope so I could use it several places (though this should definitely be a utility function in a different file), so I'll leave that for another time if someone is refactoring this code